docs/workflows/build-pages-with-storybook.md

```diff
@@ -117,13 +117,75 @@ module.exports = {
   // your Storybook configuration
 
   webpackFinal: config => {
-    config.resolve.alias.fetch = '../__mocks__/fetch.js'
+    config.resolve.alias['isomorphic-fetch'] = require.resolve('../__mocks__/isomorphic-fetch.js');
     return config;
    }
 };
 ```
 
-You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story.
+You would still need to write the fetch mock and wire up a decorator to provide results to it based on the current story. An example of how that might look:
+
+```js
+// __mocks__/isomorphic-fetch.js
+let nextJson;
+export default async function fetch() {
+  if (nextJson) {
+    return {
+      json: () => nextJson,
+    };
+  }
+  nextJson = null;
+}
+
+export function decorator(story, { parameters }) {
+  if (parameters && parameters.fetch) {
+    nextJson = parameters.fetch.json;
+  }
+  return story();
+}
+```
+
+You would then wire it up like so:
+```js
+// .storybook/preview.js
+import { decorator } from '../__mocks/isomorphic-fetch';
+
+// Add the decorator to all stories
+export const decorators = [decorator];
+```
+
+Then to set the value for a specific story you could do (to borrow an example from this [blog post](https://medium.com/@edogc/visual-unit-testing-with-react-storybook-and-fetch-mock-4594d3a281e6)):
```

Comment:
@tmeasday we probably could improve this a bit and probably say:

" Once you got all the necessary configurations done, you can set the values in any specific story like so (you can borrow from this [blog post](https://medium.com/@edogc/visual-unit-testing-with-react-storybook-and-fetch-mock-4594d3a281e6)"

WDYT?

Also as a side note, probably being a bit picky and sorry for that. Can you add the following to the code snippet below:

```js
// App.story.js

import React from 'react'
.....
```

So that the reader can get a point of reference.